src/pages/PatientsPage.tsx

```diff
@@ -5,11 +5,32 @@ import usePatients from '../hook/patient.hook'
 import useTranslationHook from '../hook/TranslationHook'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import TitlePage from '../components/TitlePage'
+import { useCallback, useState } from 'react'
+import { Patient } from '../types'
+import SearchBar from '../components/SearchBar'
 
 const PatientsPage = (): JSX.Element => {
   const { patients, reloadPatients } = usePatients()
   const { t } = useTranslationHook()
 
+  // Search Bar
+  const [filteredPatients, setFilteredPatients] = useState<Patient[]>(patients)
+
+  const filterByPatients = (patient: Patient, searchValue: string): boolean => {
+    const fullName = `${patient.lastname} ${patient.firstname}`.toLowerCase()
+    return (
+      fullName.includes(searchValue.toLowerCase()) ||
+      patient.lastname.toLowerCase().includes(searchValue.toLowerCase()) ||
+      patient.firstname.toLowerCase().includes(searchValue.toLowerCase()) ||
```

Comment:
Why do you need to search in lastname and firstname if you're already searching in fullname?

Reply:
depend on you feel you're starting the search

Reply:
yes I see